src/auth/googleauth.ts

```diff
@@ -629,6 +663,21 @@ export class GoogleAuth {
     }
   }
 
+  /**
+   * Gets the project ID from external account client if available.
+   */
+  private async getExternalAccountClientProjectId(): Promise<string | null> {
+    if (this.jsonContent && this.jsonContent.type === EXTERNAL_ACCOUNT_TYPE) {
```

Comment:
[style, optional] invert the condition to reduce nesting of the code

Reply:
Done